src/redux/create.ts

```diff
@@ -0,0 +1,21 @@
+import { combineReducers, createStore, applyMiddleware, compose } from "redux";
+import thunk from "redux-thunk";
+import appStatusReducer, { AppStatusAction } from "./modules/appStatus";
+
+export const rootReducer = combineReducers({
+  appStatus: appStatusReducer
+});
+
+export const store = createStore(
+  rootReducer,
+  compose(
+    applyMiddleware(thunk),
+    (window as any).__REDUX_DEVTOOLS_EXTENSION__
```

Comment:
instead of adding any here we can modify the window interface by adding:

`src/index.d.ts`
```
interface Window {
  __REDUX_DEVTOOLS_EXTENSION__?: () => Function;
}
```

then we can rewrite this as
```
export const store = createStore(
  rootReducer,
  compose(
    applyMiddleware(thunk),
    window.__REDUX_DEVTOOLS_EXTENSION__
      ? window.__REDUX_DEVTOOLS_EXTENSION__()
      : (f: any) => f
  )
);
```